webcompat/form.py

```diff
@@ -154,6 +159,22 @@ def normalize_url(url):
     return url
 
 
+def normalize_metadata(metadata_value):
+    """Normalize the metadata received from the form."""
+    # Removing closing comments.
+    if metadata_value is None:
+        return None
+    if '-->' in metadata_value:
+        metadata_value = metadata_value.replace('-->', '')
+        metadata_value = normalize_metadata(metadata_value)
+    # Let's avoid html tags in
+    if ('<' or '>') in metadata_value and '-->' not in metadata_value:
+            metadata_value = ''
+    if len(metadata_value) > 200:
```

Comment:
Interesting. Hadn't thought about length attacks/spam. There's also an option to truncate but I can't think of a great reason why you would have something that long in a UA string or browser name and it be worth keeping any of it. Other than accidental cmd+v or something.

Reply:
Yes I hesitated in between the two, aka truncating or empty, but as you said I don't see a lot of reason to keep something insanely long which might be misleading anyway when it's time to generate a label.